en/lessons/r-basics-with-tabular-data.md

```diff
@@ -57,9 +57,10 @@ The console is where you will type commands. To clear the initial screen, go to
 
 Before working with your own data, it helps to get a sense of how R works by using the built-in data sets. You can search through the data sets by entering `data()` into the console. This will bring up the list of all of the available data sets in a separate window. This list includes the titles of all of the different data sets as well as a short description about the information in each one.
 
-Take a look at the `AirPassengers` data set by typing `data(AirPassengers)` into the console[^1]. This will load the data set into the console. To see the data set, type in `AirPassengers` on the next line and hit *Enter*. This will bring up a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. Enter `data(AirPassengers)` into the console and hit *Enter*. On the next line, type in `AirPassengers` and hit *Enter* again. You should see:
+First, you need to load the AirPassengers data set into your R session. Type `data(AirPassengers)` and hit *Enter*[^1]. To view the data set, type in `AirPassengers` on the next line and hit *Enter* again. This will print a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. You should see:
 
 ```
+> data()
```

Comment:
Maybe we can remove this extra `data()`, as it is mention in the previous paragraph just as an example of how to take a look to all the available datasets. It is not really a step expected for users to do for what comes next

Reply:
Yes, okay. I'll remove this!